schematic/utils/df_utils.py

```diff
@@ -34,7 +34,7 @@ def load_df(file_path, preserve_raw_input=True, **kwargs):
         float_df=deepcopy(org_df)
 
         #Find integers stored as strings 
-        ints = org_df.applymap(lambda x: np.int(x) if str.isdigit(x) else False, na_action='ignore').fillna(False)
+        ints = org_df.applymap(lambda x: int(x) if str.isdigit(x) else False, na_action='ignore').fillna(False)
```

Comment:
@GiaJordan I know you switched to `np` for a reason - rolling back this particular line would work with your general approach on input going forward?